### PR TITLE
Issue #208: Add tests to run the example scripts

### LINKED ISSUE #208: Add automated testing of user facing example scripts
Add automated testing for the user facing example scripts in example_scripts folder.

These sits outside the main code in fitbenchmarking folder. These should be tested against

* that they are run with the expected software and minimizers
* that they don't fail

### PR DESCRIPTION
**Pull request by AndrewLister-STFC**
#### Description of Work

Fixes #208

#### Testing Instructions

1. Check Travis has run the additional tests
2. Check modified example scripts still run as standalone scripts

### CODE REVIEW

**Comment by AndrewLister-STFC** [comment]
Travis failed due to timeout with Mantid. Perhaps we should take some problems out of the Mantid example?

File: example_scripts/example_runScripts_mantid.py [hunk 1]
```diff
@@ -36,81 +36,86 @@
...
-                color_scale=color_scale, results_dir=results_dir)
-
-  print('\nCompleted benchmarking for {} problem set\n'.format(sub_dir))
+def main(argv):
+    # SPECIFY THE SOFTWARE/PACKAGE CONTAINING THE MINIMIZERS YOU WANT TO BENCHMARK
+    software = ["mantid"]
+    software_options = {'software': software}
+
+    # User defined minimizers
+    custom_minimizers = {"mantid": ["Simplex"],
```
Comment (tyronerees) [inline example_scripts/example_runScripts_mantid.py]: Simplex is one of the slowest methods -- if we change this to levenberg marquardt, does this fix the timeout issue?

Comment (AndrewLister-STFC) [reply example_scripts/example_runScripts_mantid.py]: I'll give it a go.

**Comment by AndrewLister-STFC** [comment]
Travis didn't start automatically with my last commit.... I've restarted a previous failed test which should use the latest version, but the red/green dot will be on the wrong commit

File: example_scripts/example_runScripts_mantid.py (hunk 1 above)
Comment (AndrewLister-STFC) [reply example_scripts/example_runScripts_mantid.py]: Not sure if its Travis not picking up the new code or if it didn't work... I think closing a PR and opening it normally kicks off a test though?

**Pull request merged**